.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js: "8.12.0"
```

Comment:
One of the advantages of using Travis CI is the ability to run the test with multiple Node versions easily.

Please test with Node `latest`, `8` and `10`.